### DISCUSSION TITLE
Introduce Vernuntii as Cake extension

### DISCUSSION THREAD

**Discussion by teneko** [cake-build/cake#3882] [Extension requests]
Hi I am library author of Vernuntii a versioning tool. I would like to know what the process is to get listed when contributing a cake addin. Are we allowed to create a Cake.* in nuget or do we need to be invited by you?

**Comment by pascalberger** [comment]
Publishing addins with a `Cake.*` prefix to NuGet is possible by everybody. If you're addin follow [best practices for authoring Cake addins](https://cakebuild.net/docs/extending/addins/best-practices) we have some automation in place which should find the addin once published to NuGet and automatically add them to the cakebuild.net website. 

If you plan to maintain the addin in a separate addin, we also have the [cake-contrib](https://cakebuild.net/community/cake-contrib) organization on GitHub, where different people are maintaining their addin using a central infrastructure.
